### DISCUSSION TITLE
RAUC and dm-verity protected partition : root=/dev/dm-0

### DISCUSSION THREAD

**Discussion by gbetous** [rauc/rauc#1449] [Q&A]
Hi,

I successfully made RAUC running on a system with root A B partitions. As far as I understood, RAUC is looking at the kernel command line the root=/dev/mmcblk0p2 (or mmcblk0p3) parameter and compare it with the description given in /etc/rauc/system.conf.

Now I want to enable dm-verity protection, the root= is always /dev/dm-0

How can I handle that ?

Thanks !

**Comment by jluebbe** [comment]
Please take a look at https://rauc.readthedocs.io/en/latest/integration.html#booted-slot-detection
